plop/component/component-ts.txt

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { ELEMENT_TYPES, getTestId } from "../../utils/test-utils";
+import styles from "./{{properCase componentName}}.module.scss";
+
+export interface {{properCase componentName}}Props extends VibeComponentProps {
+  /**
+   * class name to be added to the wrapper
+   */
+  className?: string;
```

Comment:
I think there is no need to define these props because they are already declared in VibeComponentProps. Am I right?

Reply:
U r totally right. Let's replace it with some `customProp`